sklearn/cluster/tests/test_affinity_propagation.py

```diff
@@ -181,6 +181,27 @@ def test_equal_similarities_and_preferences():
     assert _equal_similarities_and_preferences(S, np.array(0))
 
 
+def test_affinity_propagation_random_state():
+    # Significance of random_state parameter
+    # Generate sample data
+    centers = [[1, 1], [-1, -1], [1, -1]]
+    X, labels_true = make_blobs(n_samples=300, centers=centers,
+                                cluster_std=0.5, random_state=0)
+    # random_state = 0
+    ap = AffinityPropagation(convergence_iter=1, max_iter=2, random_state=0)
+    ap.fit(X)
+    centers0 = ap.cluster_centers_
+
+    # random_state = 76
+    ap = AffinityPropagation(convergence_iter=1, max_iter=2, random_state=76)
+    ap.fit(X)
+    centers76 = ap.cluster_centers_
+
+    abs_val_diff = np.sum((centers0 - centers76) * (centers0 - centers76))
+
+    assert abs_val_diff > 0.01
```

Comment:
`abs_val_diff` is about 900. I think you meant to call `mean` instead of `sum`?

The MSE is about 1.65 so we can have

```py
assert np.mean((centers0 - centers76)**2) > 1
```

which is a satisfactory threshold

Reply:
> I think you meant to call `mean` instead of `sum`

I think I was just somewhere else... :)
Suggestion applied. Thanks!